### DISCUSSION TITLE
Docker / Rancher Question

### DISCUSSION THREAD

**Discussion by Compukid** [ciur/papermerge#285] [Q&A]
I recently watched a youtuber called DBTech (https://www.youtube.com/watch?v=i25egTHFol4&t=6s) install papermerge on Docker / portainer. I am new to containers and have been trying out rancher. I was able to install papermerge on rancher with no issues and still just looking around and had some questions. 

I should just test this out, but not sure how exactly. Can I use an SMB share as the /data folder? this way all the papermerge data is in a centralized location? 
   If on a share can I just back up the share or do I still need to run a command to backup the data? 
   
   lastly Rancher support kubernetes which allows to manages multiple instances of a container making that software / app more 
   resilient. So if data is on a centralized share could I theoretically spin up multiply instances of papermerge and just point to the 
   same /data location? Or will these other instances cause some kind of contention and lockup causing instability? 

I appreciate you taking the time and looking at my questions.  I only installed your software a few days ago, but seems great. I am just using it for at home use to store financial, home and other papers that I may need to search on later. This seems like a good use case for me and maybe can think of other use cases later on. 

Thanks again

**Comment by ciur** [comment]
@Compukid, Rancher support is out of scope.
